spec/factories/division.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :division do
+    name 'Test Division'
```

Comment:
Are divisions' names unique? If not, should they be? If so, what if I create more than one in a test environment?

Reply:
I don't see any reason they shouldn't, changed this for union, position and division.